linchpin/cli/__init__.py

```diff
@@ -144,8 +144,10 @@ def _write_to_inventory(self, tx_id=None, inv_path=None, inv_format="cfg"):
                             # if its not absolute path make path
                             # relative to workspace
                             if not os.path.isabs(i_path):
-                                i_path = "{0}/{1}".format(self.ctx.workspace,
-                                                          i_path)
+                                i_path = "{0}/{1}/{2}".format(
+                                        self.ctx.get_cfg('evars','workspace'),
+                                        self.ctx.get_cfg('evars','inventories_folder'),
+                                        i_path)
```

Comment:
I actually thought this would be the `default_inventory_path`. This does provide a similar location, but seems to have missed the point. A user would now have to know that the location defaults to the values of `<workspace>/<inventories_folder>`, rather than a single value of `<default_inventory_folder>`. Which should be used from the configuration?